src/overlays/actors/ovl_Bg_F40_Block/z_bg_f40_block.c

```diff
@@ -56,7 +56,7 @@ s32 func_80BC3980(BgF40Block* this, PlayState* play) {
     this->unk_160 = 0;
     this->unk_164 = 0;
 
-    if (BGF40BLOCK_GET_PATH(&this->dyna.actor) != 0x3F) {
+    if (BGF40BLOCK_GET_PATH(&this->dyna.actor) != PATH_INDEX_MAX) {
```

Comment:
```suggestion
    if (BGF40BLOCK_GET_PATH_INDEX(&this->dyna.actor) != PATH_INDEX_MAX) {
```
Very tempted to go through and comment on some consistency things and name some actor unknowns. (I'm seeing `pathId`, `pathnum`, `pathLocation`), would that cleanup be something you're interested in? Can leave it outside the scope of this PR if you'd like and note it down for later

Reply:
I don't mind too much doing it. Give me a bit to actually go through and clean them up before you add all the reviews please :).

Reply:
Ok I think that is it for now.